### DISCUSSION TITLE
conditional rendering using custom fields

### DISCUSSION THREAD

**Discussion by daliokas7** [netbox-community/netbox#9587] [Q&A] [answered]
I'm trying to create custom link with conditional rendering which is using custom field(as per example from: https://docs.netbox.dev/en/stable/models/extras/customlink/).

My conditional custom link: {% if object.custom_fields.specifications == true %}Specifications{% endif %}
But at Device Type page I see an error 'dcim.models.devices.DeviceType object' has no attribute 'custom_fields'

I also tried to skip 'custom_fields': {% if object.specifications == true %}Specifications{% endif %} but link dissapeared altogether. As I understand, there are no such field as 'specifications' at object's root.

Is it a bug or limitation or some mistake?
![Untitled](https://user-images.githubusercontent.com/52320292/175250709-59221b2c-a95a-421b-8a7b-071d474830e6.png)

**Comment by jeremystretch** [comment, marked as answer]
You can reference custom field values on an object using [the `cf` attribute](https://docs.netbox.dev/en/stable/customization/custom-fields/#custom-fields-in-templates).

**Reply by daliokas7** [reply]
thanks!
